docs/sources/reference/commandline/cli.md

```diff
@@ -599,6 +606,28 @@ repository is used as Dockerfile. Note that you
 can specify an arbitrary Git repository by using the `git://` or `git@`
 schema.
 
+    $ sudo docker build -f Dockerfile.debug .
+
+This will use a file called `Dockerfile.debug` instead of `Dockerfile`
+during the build process. If a `Dockerfile` also exists it will be ignored.
```

Comment:
perhaps it would be better not to use the word `ignored` - as `.dockerignore` means the file isn't added to the context?

maybe 

```
This will use a file called `Dockerfile.debug` for the build instructions, instead of the `Dockerfile`.
```